game/forms.py

```
@@ -30,26 +43,31 @@ def clean(self):
-                email_template_name, 
-                context, 
-                from_email, 
+                subject_template_name,
+                email_template_name,
+                context,
+                from_email,
                 to_email,
-                html_email_template_name)
+                html_email_template_name
+            )
         except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)